src/services/dacFxService.ts

```diff
@@ -145,4 +145,8 @@ export class DacFxService implements mssql.IDacFxService {
 		};
 		return this._client.sendRequest(dacFxContracts.ValidateStreamingJobRequest.type, params);
 	}
+
+	savePublishProfile(profilePath: string, databaseName: string, connectionString: string, sqlCommandVariableValues?: Map<string, string>, deploymentOptions?: mssql.DeploymentOptions): Thenable<mssql.ResultStatus> {
+		throw new Error('Method not implemented.');
```

Comment:
@SakshiS-harma I left this as not implemented since it will be part of adding publish profile support in vscode

Reply:
Yes, I will add it later with the other publish profile changes.